packages/EasyCodingStandard/packages/Configuration/src/Exception/Guard/DuplicatedCheckersLoadedException.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\EasyCodingStandard\Configuration\Exception\Guard;
+
+use Exception;
+
+final class DuplicatedCheckersLoadedException extends Exception
```

Comment:
I was afraid of custom Exception, but man, they look beautiful :heart:

Reply:
:clap: